roles/kubernetes-apps/csi_driver/aws_ebs/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+enableVolumeScheduling: true
```

Comment:
may be add prefix csi_ to variable names ?

Reply:
True, good call!